packages/components/src/utils/breakpoints.ts

```diff
@@ -44,11 +44,20 @@ class Breakpoint {
         return breakpoint.minWidth <= innerWidth;
       });
 
+      if (!this.currentBreakpoint) {
+        return;
+      }
+
       if (!options.emitEvents) return;
```

Comment:
The two checks can be merged: 
```suggestion
      if (!this.currentBreakpoint || !options.emitEvents) return;
```